tests/Feature/IndividualTest.php

```diff
@@ -756,6 +774,52 @@
     $this->assertTrue($individual->checkStatus('draft'));
 });
 
+test('individual pages cannot be published by other users', function () {
+    $user = User::factory()->create();
+    $individual = Individual::factory()->create([
+        'roles' => ['consultant'],
+        'published_at' => null,
+    ]);
+
+    $response = $this->actingAs($user)->from(localized_route('individuals.show', $individual))->put(localized_route('individuals.update-publication-status', $individual), [
+        'publish' => true,
+    ]);
```

Comment:
@greatislander I tried removing the `from` portion, as per our conversation but it still wasn't hitting the publish policy. I believe the `put` will actually fail on the `update` policy from the 'individuals.update-publication-status' route. So I'm not sure it's actually possible to hit the line of code. I'm going to collapse the policy back down and for now revert back to the old message as it's more generic in case something does slip through.